sycl/test/check_device_code/group_ballot.cpp

```diff
@@ -3,9 +3,6 @@
 
 #include <sycl/sycl.hpp>
 
-int main() {
-  sycl::queue Q;
-  Q.parallel_for(sycl::nd_range<1>{32, 32}, [=](sycl::nd_item<1> item) {
-    auto Mask = sycl::ext::oneapi::group_ballot(item.get_sub_group());
-  });
+SYCL_EXTERNAL void cast_group_ballots(sycl::nd_item<1> item) {
```

Comment:
This test should also be moved out of this folder.

Reply:
Should the group tests have their own group folder? Or does it just not belong in check_device_code?

Reply:
The latter - LLVM IR isn't verified in any way in this test.

Reply:
I'd start with `sycl/test/extensions/group_ballot.cpp`